app/client/components/navbar/desktop/TopNavDropdownLink.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ComponentPropsWithoutRef, forwardRef } from "react";
+import { PropsWithChildren } from "react";
```

Comment:
These imports should be merged as they come from the same source.

Please check it in other files as well.

Reply:
Done.

I've added a prettier plugin for organizing imports. Now you can run `npm run format` and it will format all files